add submission form (#24)
FAIRmat-NFDI/nomad-gallery: docs/submissions.md

Reviewer: ```suggestion
 In case of any question you can contact us via our [Discord server](https://discord.gg/WYBF8S6y2Z) or [Github](https://github.com/FAIRmat-NFDI/nomad-gallery/issues).

Once submitted, a team member will review your entry to make sure everything looks good and will take care of publishing it. You will receive a confirmation once your submission has been processed.

If you have any questions, feel free to reach out through our [Discord server](https://discord.gg/WYBF8S6y2Z) or write to fairmat@physik.hu-berlin.de.
```

Author: can you perhaps double check the final `submissions.md`, I might have got lost because there're a few patches on this, thanks

**Submission**
Start by creating a Markdown file following this template to ensure your contribution meets the required format.

> Start by filling the [Submission Form Template](https://github.com/FAIRmat-NFDI/nomad-gallery/issues/new?template=submission.yml).

Reviewer: ```suggestion
**We welcome contributions to the gallery!**
To submit a new entry, simply fill out the [Submission Form](https://github.com/FAIRmat-NFDI/nomad-gallery/issues/new?template=submission.yml)
The form will guide you through the required information to ensure your submission follows the 
```

Author: can you double check this? it looks incomplete after "to ensure your submission follows the ?"

> **Publication**

Reviewer: ```suggestion

```
